variants/meshlink/platformio.ini

```diff
@@ -0,0 +1,128 @@
+; The very slick RAK wireless RAK 4631 / 4630 board - Unified firmware for 5005/19003, with or without OLED RAK 1921
+[env:meshlink]
+extends = nrf52840_base
+board = meshlink
+;board_check = true
+build_flags = ${nrf52840_base.build_flags} -I variants/meshlink -D MESHLINK
+  -L "${platformio.libdeps_dir}/${this.__env__}/bsec2/src/cortex-m4/fpv4-sp-d16-hard"
+  -D GPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+  -D EINK_DISPLAY_MODEL=GxEPD2_213_B74
+  -D EINK_WIDTH=250
+  -D EINK_HEIGHT=122
+  -D USE_EINK_DYNAMICDISPLAY            ; Enable Dynamic EInk
+  -D EINK_LIMIT_FASTREFRESH=5           ; How many consecutive fast-refreshes are permitted
+  -D EINK_LIMIT_RATE_BACKGROUND_SEC=30  ; Minimum interval between BACKGROUND updates
+  -D EINK_LIMIT_RATE_RESPONSIVE_SEC=1   ; Minimum interval between RESPONSIVE updates
+  -D EINK_LIMIT_GHOSTING_PX=2000        ; (Optional) How much image ghosting is tolerated
+  -D EINK_BACKGROUND_USES_FAST         ; (Optional) Use FAST refresh for both BACKGROUND and RESPONSIVE, until a limit is reached.
+  -D EINK_HASQUIRK_VICIOUSFASTREFRESH   ; Identify that pixels drawn by fast-refresh are harder to clear
+
+
+build_src_filter = ${nrf52_base.build_src_filter} +<../variants/meshlink>
+lib_deps = 
+  ${nrf52840_base.lib_deps}
+  https://github.com/meshtastic/GxEPD2#55f618961db45a23eff0233546430f1e5a80f63a
+  melopero/Melopero RV3028@^1.1.0
+  rakwireless/RAKwireless NCP5623 RGB LED library@^1.0.2
+  beegee-tokyo/RAKwireless RAK12034@^1.0.0
+debug_tool = jlink
+
+
+
+; If not set we will default to uploading over serial (first it forces bootloader entry by talking 1200bps to cdcacm)
+; Note: as of 6/2013 the serial/bootloader based programming takes approximately 30 seconds
+;upload_protocol = jlink
+
+; Allows programming and debug via the RAK NanoDAP as the default debugger tool for the RAK4631 (it is only $10!)
+; programming time is about the same as the bootloader version.
+; For information on this see the meshtastic developers documentation for "Development on the NRF52"
+[env:meshlink_dbg]
```

Comment:
Do you actually use the debug target? or is this just copied from the 4631 define? If it is just copied, please remove, as it clutters up the environments.